src/main/resources/copy_on_click.js

```diff
@@ -1,19 +1,15 @@
-/* https://stackoverflow.com/a/45071478 */
-/* doesn't work yet */
-const items = document.querySelectorAll("span");
-
-items.forEach(item => {
-  item.onclick = function() {
-    item.select();
-    item.setSelectionRange(0, 1048576);
-    document.execCommand("copy");
-  };
-});
-
-items.forEach(item => {
-  item.addEventListener("copy", function(event) {
-    event.preventDefault();
-    event.clipboardData.setData("text/plain", span.textContent);
-    console.log(event.clipboardData.getData("text"));
-  })
-});
+document.onreadystatechange = function() {
+  if(document.readyState == "complete") {
+    const items = document.querySelectorAll(".copyable");
+    items.forEach(item => {
```

Comment:
Need to make sure this is only added once per page, otherwise this may add the click handler to the cell more than once.

Reply:
Hmm, I add this js block once per java member (class, method, field). Might need to figure out a way to add this only once on each page, though not sure if taglet can detect that.

Reply:
> Need to make sure this is only added once per page, otherwise this may add the click handler to the cell more than once.

Well, as mentioned, this specific code is prepared to only run once since it sets the `onreadystatechange` event in a way that only supports one event.

> Hmm, I add this js block once per java member (class, method, field). Might need to figure out a way to add this only once on each page, though not sure if taglet can detect that.

Don't know again if possible, but this could just be added once per page (maybe a reference to a file so it can be cached/reused) regardless of whether a table is present. It will only affect `.copyable` elements, only running when page is fully loaded, so it wouldn't be a problem if it was present in other pages or at arbitrary positions.